soc/riscv/riscv-privilege/opentitan/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_sources(soc.c rom_header.S)
+zephyr_compile_options(-march=rv32imc_zicsr_zifencei)
```

Comment:
This should not be explicitly set, instead the SoC should select `RISCV_ISA_EXT_ZICSR` or `RISCV_ISA_EXT_ZIFENCEI`, explicitly or implicitly by selecting for example `RISCV_ISA_EXT_G`. See `arch/riscv/Kconfig.isa`

Reply:
Switched to implicit selection Kconfig.soc.